lib/settings.coffee

```diff
@@ -37,6 +37,10 @@ module.exports = new Settings 'vim-mode-plus',
     items: type: 'string'
     default: []
     description: 'Start in insert-mode whan editorElement matches scope'
+  clearMultipleCursorsOnEscapeInsertMode:
```

Comment:
I think this is going to help a lot of people out. Will this also apply to visual block selection?